### PR TITLE
[#1123] Changes position and text of "All Organizations" button

### LINKED ISSUE #1123: fix in position and text of all organization button
**Is your feature request related to a problem? Please describe.**
All organization button is not intutive

**Describe the solution you'd like**
Position the all organization button at the top of the menu (i.e above dashboard button) and change all organization text to my organizations

**Additional context**
Screenshot 2023-12-04 at 11 09 09 PM

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by adi790uu**
**What kind of change does this PR introduce?**

This PR renames the "All Organizations" button to "My Organizations" and fixes its postion as specified in the issue.

**Issue Number:**

Fixes #1123 

**Did you add tests for your changes?**

No

**Snapshots/Videos:**

[Screencast from 05-12-23 03:14:37 PM IST.webm](https://github.com/PalisadoesFoundation/talawa-admin/assets/132701661/543631e7-37ba-4527-a0f6-be5c8ccda093)

**If relevant, did you update the documentation?**

No

**Summary**

This pull request addresses the specified issue by renaming the "All Organizations" button to "My Organizations" and adjusting its position to enhance the UI/UX.

**Does this PR introduce a breaking change?**

No

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**

Yes

### CODE REVIEW

**Comment by aashimawadhwa** [comment]
Please share a screenshot and  
Please make the code coverage 100% for the modified file. It has been declining overall and we need to make sure we get it back up to make our code more stable.
@adi790uu

**Review by noman2002 — CHANGES_REQUESTED** [review]
Please fix the failing test.

**Comment by adi790uu** [comment]
Ok will work on it!

**Comment by noman2002** [comment]
The coverage is dropping significantly. Please get the coverage to 100% for the files you edited and merge the latest code into this PR.

**Comment by adi790uu** [comment]
@aashimawadhwa @beingnoble03  The coverage is down because of files which are not under the scope of this pull request and issue, what should I do from here?

File: src/GraphQl/Queries/Queries.ts
Comment (noman2002) [inline src/GraphQl/Queries/Queries.ts:4]: Unnecessary change.

**Review by noman2002 — CHANGES_REQUESTED** [review]
Remove the mentioned unnecessary changes.

File: src/components/OrganizationScreen/OrganizationScreen.tsx
Comment (noman2002) [inline src/components/OrganizationScreen/OrganizationScreen.tsx:16]: Unnecessary change.

**Review by aashimawadhwa — APPROVED** [review]

**Review by palisadoes — APPROVED** [review]

**Review by aashimawadhwa — APPROVED** [review]

**Comment by aashimawadhwa** [comment]
@palisadoes  this can be merged

**Review by palisadoes — APPROVED** [review]

**Comment by palisadoes** [comment]
@noman2002 please review. We can't merge until you have dismissed your previous requests for change.

**Review by noman2002 — APPROVED** [review]

**Comment by noman2002** [comment]
@adi790uu Need to revert this. This should be develop branch not on main.

**Pull request merged**
